src/frame/headers.rs

```diff
@@ -844,6 +852,46 @@ impl HeaderBlock {
             .map(|(name, value)| decoded_header_size(name.as_str().len(), value.len()))
             .sum::<usize>()
     }
+
+    /// Iterator all pseudos and headers to see if any individual pair
```

Comment:
Nit: I think there's a typo here:
```suggestion
    /// Iterate over all pseudos and headers to see if any individual pair
```